python/sqlcommenter-python/google/__init__.py

```diff
@@ -1,17 +0,0 @@
-#!/usr/bin/python
-#
-# Copyright The OpenTelemetry Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
any reason to not add this line in `opentelemetry/__init__.py` ?

Reply:
Accidentally removed. Added now